Documentation/guides/BuildProcess.md

```diff
@@ -1135,6 +1135,29 @@ server, the following MSBuild properties can be used:
 
     Note: `env:` is not supported when using `AndroidPackageFormat`=`aab`. 
 
+-   **JarsignerTimestampAuthorityUrl** &ndash; This property
+    allows you to specify a Url to a timestamp authority
+    service. This can be used to make sure your apk signature
+    includes a timestamp.
+    See [time-of-signing](https://docs.oracle.com/javase/8/docs/technotes/guides/security/time-of-signing.html) for more details.
+
+    ```xml
+    <PropertyGroup>
+        <JarsignerTimestampAuthorityUrl >http://example.tsa.url</JarsignerTimestampAuthorityUrl>
+    </PropertyGroup>
+    ```
+
+-   **JarsignerTimestampAuthorityCertificateAlias** &ndash; This 
+    property allows you to specify an alias in the keystore
+    for a timestamp authority.
+    See [time-of-signing](https://docs.oracle.com/javase/8/docs/technotes/guides/security/time-of-signing.html) for more details.
```

Comment:
Just throw out an alternative wording option in case it might be useful, one idea could be to use the article title, like:

```
See the Java [Signature Timestamp Support](https://docs.oracle.com/javase/8/docs/technotes/guides/security/time-of-signing.html) documentation for more details.
```